### ISSUE TITLE
Missing F2FS support for Manta and Mako

### ISSUE THREAD

**Issue by frontalot** [TeamWin/Team-Win-Recovery-Project#372]
Can you add F2FS support to both the manta and mako builds?

Thanks.

**Comment by Cirooo** [comment]
Agree, and my mako now uses an unofficial twrp version with f2fs support from xda forum. If twrp official offer the f2fs support, I think the ROM developers would consider to add the kernel level support. Any way, I use a custom kernel with f2fs support from xda forum too. 
f2fs: /data /cache . I tried  convert to /system then LineageOS format back to ext4
ROM: LineageOS
kernel: SINAI-N4
